test/unit/org/apache/cassandra/db/compaction/CompactionControllerTest.java

```diff
@@ -206,6 +206,8 @@ public void testTombstonesNotPurgedWhenRebuilding()
         StorageService.instance.unsafeSetRebuilding(true);
 
         assertEquals(CompactionController.getFullyExpiredSSTables(cfs, compacting, overlapping, gcBefore), Collections.emptySet());
+
+        StorageService.instance.unsafeSetRebuilding(true);
```

Comment:
should this be false? can also do a try finally so if this test fails it doesn't affect other ones then too

Reply:
oops yes, changed to try finally